docs/notebook_tutorial/tutorial_eliza.ipynb

```diff
@@ -0,0 +1,169 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Eliza Example\n",
+    "\n",
+    "This example will walk you through the steps to set up a Eliza chatbot which generates responses based on Eliza rules. We will utilize `stave` to visualize the dialogue page that allows you to chat with the bot interactively.\n",
```

Comment:
`an Eliza`

Reply:
Probably can say two things:
1. This tutorial is to help one to understand how visualization work together with the pipeline
2. Add citation or just the wiki link for Eliza